cpp/modmesh/toggle/pymod/wrap_profile.cpp

```diff
@@ -181,6 +181,11 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapCallProfiler : public WrapBase<WrapC
                 "instance",
                 [](py::object const &) -> wrapped_type &
                 { return wrapped_type::instance(); })
+            .def("stat", [](CallProfiler & profiler)
+                 {
+                std::stringstream ss;
+                profiler.print_statistics(ss);
+                return ss.str(); })
             .def("result", [](CallProfiler & profiler)
                  {
             const RadixTreeNode<CallerProfile> * root = profiler.radix_tree().get_root();
```

Comment:
The indentation is off, but it is not a major issue needing to be fixed in this PR.

The function is too long to be reasonably placed in a lambda.  Please replace the lambda with a static function in this wrapper class.  You can reference the [wrapping code for `SimpleArray.__setitem__()`](https://github.com/solvcon/modmesh/blob/70a033705854537bbb2329d6571cf125eeac495a/cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp#L134):

```cpp
            .def("__setitem__", &property_helper::setitem_parser)
```

But you do not need to use a separate helper class like `property_helper`.